src/app/app.component.html

```diff
@@ -1,8 +1,13 @@
 <h1>Scroll down ↓</h1>
+<h3>Placeholder image loaded: <span class="placeholder-boolean">{{ placeholderLoaded }}</span></h3>
 <div class="spacer"></div>
+<h3>fullResLoaded event count: <span class="full-res-count">{{ fullResLoadedEventCount }}</span></h3>
 <sn-image-loader
   [image]="image"
   [sizes]="sizes"
   imgClass="foo"
+  alt="lorem ipsum"
+  (placeholderLoaded)="onPlaceholderLoad($event)"
+  (fullResLoaded)="onFullResLoad($event)"
```

Comment:
I would be very clear and specific and call these events `imageLoaded` and `imagePlaceholderLoaded` as `fullResLoaded` sounds ambiguous

Reply:
Changed